### PR TITLE
BZ1878469: Corrected note in BareMetal UPI Install

### PR DESCRIPTION
**Pull request by skrthomas**
https://bugzilla.redhat.com/show_bug.cgi?id=1878469

For 4.5+

Preview: https://deploy-preview-33533--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-approve-csrs_installing-bare-metal

### CODE REVIEW

File: modules/installation-approve-csrs.adoc [hunk 1]
```diff
@@ -107,7 +107,7 @@ endif::ibm-z,ibm-z-kvm[]
 +
 [NOTE]
 ====
-Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. After you approve the initial CSRs, the subsequent node client CSRs are automatically approved by the cluster `kube-controller-manager`.
+Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. Once the client CSR is approved, the Kubelet creates a secondary CSR for the serving certificate, which requires manual approval. Then subsequent serving certificate renewal requests are automatically approved by the cluster-machine-approver if the Kubelet requests a new certificate with identical parameters.
```
Comment (ahardin-rh) [inline modules/installation-approve-csrs.adoc:110]: Then,

**Comment by ahardin-rh** [comment]
One minor suggestion + one question

File: modules/installation-approve-csrs.adoc (hunk 1 above)
Comment (skrthomas) [reply modules/installation-approve-csrs.adoc]: Yes, it should just be `machine-approver` in back ticks. Good catch ty!

**Pull request merged**